modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,612 @@
+----
++
+.Example output
+[source,terminal]
+----
+Member 7a8197040a5126c8 removed from cluster b23536c33f2cdd1b
+----
+
+.. View the member list again and verify that the member was removed:
++
+[source,terminal]
+----
+sh-4.2# etcdctl member list -w table
+----
++
+.Example output
+[source,terminal]
+----
++------------------+---------+--------------------+---------------------------+---------------------------+-------------------------+
+| ID               | STATUS  | NAME                      | PEER ADDRS                  | CLIENT ADDRS                | IS LEARNER |
++------------------+---------+--------------------+---------------------------+---------------------------+-------------------------+
+| 7a8197040a5126c8 | started | openshift-control-plane-2 | https://192.168.10.11:2380/ | https://192.168.10.11:2379/ | false |
+| 8d5abe9669a39192 | started | openshift-control-plane-1 | https://192.168.10.10:2380/ | https://192.168.10.10:2379/ | false |
++------------------+---------+--------------------+---------------------------+---------------------------+-------------------------+
+----
++
+You can now exit the node shell.
++
+[IMPORTANT]
+====
+After you remove the member, the cluster might be unreachable for a short time while the remaining etcd instances reboot.
+====
+
+. Remove the old secrets for the unhealthy etcd member that was removed.
+
+.. List the secrets for the unhealthy etcd member that was removed.
++
+[source,terminal]
+----
+$ oc get secrets -n openshift-etcd | grep openshift-control-plane-2
+----
+Pass in the name of the unhealthy etcd member that you took note of earlier in this procedure.
++
+There is a peer, serving, and metrics secret as shown in the following output:
++
+
+[source,terminal]
+----
+etcd-peer-openshift-control-plane-2             kubernetes.io/tls   2   134m
+etcd-serving-metrics-openshift-control-plane-2  kubernetes.io/tls   2   134m
+etcd-serving-openshift-control-plane-2          kubernetes.io/tls   2   134m
+----
+
+.. Delete the secrets for the unhealthy etcd member that was removed.
+
+... Delete the peer secret:
++
+[source,terminal]
+----
+$ oc delete secret etcd-peer-openshift-control-plane-2 -n openshift-etcd
+
+secret "etcd-peer-openshift-control-plane-2" deleted
+----
+
+... Delete the serving secret:
++
+[source,terminal]
+----
+$ oc delete secret etcd-serving-metrics-openshift-control-plane-2 -n openshift-etcd
+
+secret "etcd-serving-metrics-openshift-control-plane-2" deleted
+----
+
+... Delete the metrics secret:
++
+[source,terminal]
+----
+$ oc delete secret etcd-serving-openshift-control-plane-2 -n openshift-etcd
+
+secret "etcd-serving-openshift-control-plane-2" deleted
+----
+
+. Delete the control plane machine.
++
+If you are running installer-provisioned infrastructure, or you used the Machine API to create your machines, follow these steps. Otherwise, you must create the new master using the same method that was used to originally create it.
```

Comment:
Similar to an earlier comment: is there a way we can avoid using the term "master"?